src/qt/walletview.h

```diff
@@ -48,7 +48,7 @@ class WalletView : public QStackedWidget
         The wallet model represents a bitcoin wallet, and offers access to the list of transactions, address book and sending
         functionality.
     */
-    void setWalletModel(WalletModel *walletModel);
+    void setWalletModel(WalletModel& walletModel);
```

Comment:
Please don't change the `setWalletModel()` interface. It is supposed to be consistent over all objects, and setting a `NULL` model is meaningful: it should be possible to use the view classes without a model for simple testing.